src/tagging-compliance/resource_parsing.py

```diff
@@ -111,4 +111,4 @@ def lambda_handler(event, context):
     push_to_gateway(
         os.environ["prometheus_ip"], job="TaggingResourceList", registry=registry
     )
-    return {"statusCode": 200, "body": "success"}
+    return {"statusCode": 200, "body": "success"}
```

Comment:
Add EOF

Reply:
added in d4d700c790b5926a49486872d2b4441380703fa1